Update 2025-02-21-ppw-hardcore-friday.md (#1177)
tpwres/teczki-polskiego-wrestlingu: content/e/ppw/2025-02-21-ppw-hardcore-friday.md

2 = { path = "blady-jasko-akcyjniak.jpg", caption = "Graphic announcing [Johnny Blade](@/w/johnny-blade.md).", source = "Official PpW Facebook" }
3 = { path = "vodka-match.jpg", caption = "Graphic announcing [Johnny Blade](@/w/johnny-blade.md) vs [Marco Hammers](@/w/marco-hammers.md). Distorted view further emphasizes that alcohol will be involved", source = "Official PpW Facebook" }
+++
The show was announced in the closing segment of [Gruba Przesada](@/e/ppw/2025-01-25-ppw-gruba-przesada.md), with tickets immediately available for purchase at the merch table.

## Build-up
* On February 10th, PpW posted a video of [Marco Hammers](@/w/marco-hammers.md) challenging [Johnny Blade](@/w/johnny-blade.md) for a match with any stipulation of Blade's choosing - except for his signature Hardcore Match.
* The match was confirmed the following day in PpW's social media.
* [Mister Z](@/w/mister-z.md) also confirmed that while Johnny's suspension is lifted, [Biesiad Strong](@/w/biesiad.md) is still suspended and he is going to get something "special" at the show.

> * On February 13th, Johnny Blade posted a video of himself tending bar at [2Koła](@/v/2kola.md) - his day job, now openly acknowledged in PpW storylines. While pouring vodka shots, he presents his choices of stipulation - either a _Drunken Ambition Match_, or a _Gra w melanż_. The rules are simple - whoever grabs the rope, needs to drink a shot of vodka. Later in the day PpW confirmed the match, officially naming the stipulation _Gra w melanż - wódka mecz_ (roughly "Party Game - Vodka Match")

Reviewer: it's not an either-or, which suggests the two are different

```suggestion
* On February 13th, Johnny Blade posted a video of himself tending bar at [2Koła](@/v/2kola.md) - his day job, now openly acknowledged in PpW storylines. While pouring vodka shots, he presents his choices of stipulation - a _Drunken Ambition Match_, also described as _Gra w melanż_. The rules are simple - whoever grabs the rope, needs to drink a shot of vodka. Later in the day PpW confirmed the match, officially naming the stipulation _Gra w melanż - wódka mecz_ (roughly "Party Game - Vodka Match")
```

* [Cagematch event page](https://www.cagematch.net/?id=1&nr=418109)
* [Hammers challenges Blade](https://www.instagram.com/p/DF51JqMMH89/)
* [Hammers vs Blade Confirmed](https://www.facebook.com/photo/?fbid=1163483015782089&set=a.499910772139320)
* [Blade chooses stipulation](https://www.instagram.com/p/DGAqsMAC4_D/)

> * [Vodka match confirmed]

Reviewer: needs the link to follow
